app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    render '/users/index'
+  end
+
+  def show
+    render 'users/show'
+  end
+end
```

Comment:
- You have done well so far, but according to the project requirements, you are required to setup corresponding views for each of your controller actions. please fix this 👍

Reply:
noted, resolved it.